module/raid/run.py

```diff
@@ -35,6 +73,12 @@ def run(self, name='', mode='', total=0):
         if not name or not mode:
             raise ScriptError(f'RaidRun arguments unfilled. name={name}, mode={mode}')
 
+        if mode == 'ex':
+            self.ui_goto_main()
+            Reward(self.config, self.device).reward_mission(
+                daily=self.config.Reward_CollectMission,
+                weekly=False)
```

Comment:
Too many `Reward` runs. Move to `RaidDaily`, check once before running EX dailies

Reply:
I had inserted into here so that it was applicable to both tasks `Raid and RaidDaily`. But I think you are right, if user is running `Raid` themselves then its assumed they collected have the tickets themselves, no need to have alas retrieve them.  But shouldn't be after `RaidDaily` since tickets can only be acquired after doing 10+ runs

Another idea I had  a new task `RaidDailyEX`. Order would be `RaidDaily > self.config.task_call('Reward') > RaidDailyEX`. A little longer I think it would take but more naturally calls `Reward` this way.

Can't call `task_call` and have steps after it since raises an exception. I believe but if separated as two tasks it would work I think.